src/ResourceManager/EventHub/Commands.EventHub/Cmdlets/DRConfig/SetAzureEventHubDRConfigurationBreakPairing.cs

```diff
@@ -0,0 +1,50 @@
+﻿// ----------------------------------------------------------------------------------
```

Comment:
General comment:  It seems like you are missing some piping scenarios: https://github.com/Azure/azure-powershell/wiki/Piping-in-PowerShell#using-the-resourceid-parameter.